requirements/run.in

```diff
@@ -2,7 +2,7 @@
 # pin versions outside pypi, i.e. latest git master branch version.
 ipython
 lockfile
-python-openflow>=2021.1
+python-openflow>=2022.1rc1
```

Comment:
@ajoaoff this looks great, but it'll probably result in pip install issues since we don't have the ownership on pypi yet to tag this one yet, right? I wonder if we should leave this as just  `python-openflow`, and then `python-openflow` is installed from master on our docker file, that way when it tries to install it'll detect python-openflow has already been installed, wdyt? I'll try this out with this branch

Reply:
I've just quickly tried it out locally, fyi, without the pinned version, the docker build worked, when it [hits this line in the run.txt](https://github.com/kytos-ng/kytos/blob/master/requirements/run.txt#L32), it would detect that it's been already installed:

```
Requirement already satisfied: python-openflow in /usr/local/lib/python3.7/dist-packages (from kytos==2022.1rc1) (2021.1)
```
Also, for NApps and their `requirements/dev.txt`, even though `python-openflow` will resolve to the lastest upstream version, since most of them will install `-e` from `master`, when it'd also overwrite correctly (`pip install -r requirements/dev.txt` on mef_eline` for example):

```
  Attempting uninstall: python-openflow
    Found existing installation: python-openflow 2021.1
    Uninstalling python-openflow-2021.1:
      Successfully uninstalled python-openflow-2021.1
  Running setup.py develop for python-openflow
```